.eslintrc

```diff
@@ -54,6 +54,12 @@
       }
     ],
     "no-use-before-define": "off",
+    "object-curly-newline": [
```

Comment:
We'll want to update `eslint-config-curology` eventually, and/or add `eslint-plugin-prettier` since `--fix` overwrites our `prettier` rules, which typically occurs with 4 imports being broken into multiple lines even though the character limit does not exceeded our `prettier` settings.